freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/model/BaseSyncOperationResponse.java

```diff
@@ -0,0 +1,92 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.user.model;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import com.sequenceiq.freeipa.api.v1.freeipa.user.doc.UserModelDescriptions;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class BaseSyncOperationResponse<D> {
```

Comment:
I'll take a look at the ApiCommand and see. I'm using this for set password and user sync right now and I was expecting the responses to diverge as we add more details. I'll see if I can make the model work for both. then I can create a single operation status api instead of having three

Reply:
combined user sync and set password to have a common response and added a single getSyncOperationStatus endpoint for polling both